app/controllers/admin/communities_controller.rb

```diff
@@ -266,6 +285,8 @@ def update_settings
 
     maybe_update_payment_settings(@current_community.id, params[:community][:automatic_confirmation_after_days])
 
+    MarketplaceService::API::Marketplaces.update_marketplace_configurations(community_id: @current_community.id, main_search: params[:main_search]) if feature_enabled?(:location_search)
```

Comment:
So, you would like to create another API file, which would be called from the current one (MarketplaceService::API::Marketplaces) when marketplace is created?

I feel that we are going beyond what we planned with this micro service approach, if we create own APIs for every db tables (and we already have a store layer inside services, which makes this several-tables-behind-unified-api approach possible). 

How about reducing repetition just a bit:
MarketplaceService::API::Marketplaces.update_marketplace_configurations
=> MarketplaceService::API::Marketplaces.update_configurations

Unless, someone thinks this configuration is going to bloat that existing API? Current future plan is to add km/miles selection there too at some point, but that should not add any additional functions to API.